core/chain-configs/src/client_config.rs

```diff
@@ -134,12 +134,15 @@ pub struct GCConfig {
 }
 
 impl Default for GCConfig {
+    // Garbage Collection should be faster than the block production. As a rule
+    // o thumb it should be set to be two times faster, plus a small margin. At
+    // the current min block time of 600ms that means 2 blocks per 500ms.
```

Comment:
Should we follow-up to derive it from the configs automatically so that next time block time changes we update gc step period as well? (Or we can have tests that would fail if one changes without the other adjusted.)